src/Backend/Bigquery/ToFinalTable/SqlBuilder.php

```diff
@@ -135,6 +115,55 @@ public function getInsertAllIntoTargetTableCommand(
                 );
             }
         }
+        return $columnsSetSql;
+    }
+
+    public function getTableExistsCommand(string $dbName, string $tableName): string
+    {
+        return sprintf(
+            'SELECT COUNT(*) AS count FROM %s.INFORMATION_SCHEMA.TABLES WHERE table_name = %s;',
```

Comment:
Narazil som na to u seba ze to vrati aj views takze by sa mohlo stat ze tam bude view s menom tabulky a tabulka nebude existovat a vratilo by to ze existuje pridal by som tam `WHERE `table_type` != \'VIEW\';` pretoze BQ podporuje viac druhov tabuliek takze nestaci len `=TABLE`

Reply:
:D to sem asi enom přesunul při reformatu, tos přidával ty :D

Reply:
jo tak ja to fixujem v dalsom PR potom teda :)